### ISSUE TITLE
error with getSpectraViewerAsBlob

### ISSUE THREAD

**Issue by NRayya** [cheminfo/nmrium#2046]
Using this file and many others: http://www.nmrshiftdb.org/download/NmrshiftdbServlet/60004137_H,H-COSY.zip?spectrumid=60004137&nmrshiftdbaction=exportspec&format=rawdata

I am receiving this error: 
<img width="524" alt="image" src="https://user-images.githubusercontent.com/82588017/211003304-41ca459b-c261-4e6e-b12a-cac3891aafd1.png">

The error is happening in nmrXiv

**Comment by lpatiny** [comment]
Did you check the content of the file ?

Because when I try curl I get this 

```
~ % curl "http://www.nmrshiftdb.org/download/NmrshiftdbServlet/60004137_H,H-COSY.zip\?spectrumid\=60004137\&nmrshiftdbaction\=exportspec\&format\=rawdata" 
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

**Comment by lpatiny** [comment]
I finally succeeded to get the zip file (the browser was refusing to download from non https server). The file does not contain the data after FT and this is currently not supported. there is the ser but not 2rr file.

![image](https://user-images.githubusercontent.com/1484241/211146048-88c3b13e-de4a-4d01-9b82-7a206df2f949.png)

However in the future version the Ser file should be loaded but it actually does not work. I opened another issue to solve this.

https://github.com/cheminfo/nmrium/issues/2053
